ui/react-ui/src/index.css

```diff
@@ -0,0 +1,21 @@
+@import 'tailwindcss';
+
+.icon-hover {
+  @apply hover:text-[#1AB394] transition-all duration-300 cursor-pointer;
+}
+
+.custom-border {
+  @apply relative;
+}
+
+.custom-border::after {
+  @apply content-[''] absolute bottom-0 left-0 w-full h-0.5 bg-gray-200 z-1;
+}
+
+.border-active {
+  @apply relative;
+}
+
+.border-active::after {
+  @apply content-[''] absolute bottom-0 left-0 w-full h-0.5 bg-[#1AB394] z-2;
+}
```

Comment:
There don't seem to be any apparent issues related to syntax or structure that could cause the code to break, though I would appreciate if there's an opportunity for improvement in its aesthetics/flow/conciseness (e.g. breaking it into smaller blocks). It's also great to see use of Tailwind CSS! Let me know what changes you'd like me to make in this snippet.

Would you need help with a specific component?